great_expectations/expectations/core/expect_column_unique_value_count_to_be_between.py

```diff
@@ -208,12 +208,16 @@ class ExpectColumnUniqueValueCountToBeBetween(ColumnAggregateExpectation):
     success_keys = (
         "min_value",
         "max_value",
+        "strict_min",
+        "strict_max",
     )
 
     args_keys = (
         "column",
         "min_value",
         "max_value",
+        "strict_min",
+        "strict_max",
```

Comment:
I think `args_keys` only matters for the `Validator` workflow. Possibly only for auto-complete.